sdk/src/main/scala/com/stratio/sparkta/sdk/Output.scala

```diff
@@ -108,37 +111,44 @@ abstract class Output(keyName: String,
     }
 
   protected def getTableSchemaFixedId(tbSchema: TableSchema): TableSchema = {
-    var tableName = tbSchema.tableName.split(Output.SEPARATOR)
-      .filter(name => !fixedBuckets.contains(name) && name != timeName).mkString(Output.SEPARATOR)
-    var fields = tbSchema.schema.fields.toSeq.filter(field =>
-      !fixedBuckets.contains(field.name) && field.name != timeName)
+    var tableName = tbSchema.tableName.split(Output.Separator)
```

Comment:
Pls mark this method to a refactor and open the related issue, I'm sure that we can remove this vars